src/white.jl

```diff
@@ -102,11 +108,11 @@ BreuschPaganTest(X, e) = WhiteTest(X, e, type = :linear)
 
 testname(t::WhiteTest) = "White's (or Breusch-Pagan's) test for heteroskedasticity"
 
-population_param_of_interest(t::WhiteTest) = ("T*R2", 0, t.lm)
+population_param_of_interest(t::WhiteTest) = ("T*R2", 0, round(t.lm,digits=4))
```

Comment:
Currently we don't round value in other tests, so better don't round here either. It would probably make sense to apply some rounding when printing, but that should be applied to all tests at the same time, in another PR.

Reply:
Thanks. I have skipped the round, except for testing `pvalue` which seems has a  built-in `round(z,digits=4)`. I also fixed the indentation of that """...""" string.